### PR TITLE
Make site container span 100% width

### PR DESCRIPTION
**Pull request by finn00dev**

### CODE REVIEW

File: www/html/ci/application/views/navbar.html [hunk 1]
```diff
@@ -7,17 +7,17 @@
 {% block head %}{% endblock %}
 </head>
 <body>
-<div class="container kt-tabs-component">
```
Comment (ktorii) [inline www/html/ci/application/views/navbar.html]: Why did you remove the class here? Aren't we losing all the other css this div used to have?

File: www/html/ci/application/controllers/Landing.php [hunk 2]
```diff
@@ -10,15 +10,15 @@ public function __construct() {
 		parent::__construct();
 
 		// load database
-        $this->load->database();
+//        $this->load->database();
```
Comment (ktorii) [inline www/html/ci/application/controllers/Landing.php]: This file shouldn't have any changes

File: www/html/ci/application/views/navbar.html [hunk 3]
```diff
@@ -7,17 +7,17 @@
 {% block head %}{% endblock %}
 </head>
 <body>
-<div class="container kt-tabs-component">
+<div style="width:100%;">
```
Comment (ktorii) [inline www/html/ci/application/views/navbar.html]: Inlined css should be avoided

**Review by ktorii — CHANGES_REQUESTED** [review]

**Comment by finn00dev** [comment]
Okay, Fixed the issues and commited them

File: www/html/css/index2.css [hunk 4]
```diff
@@ -1,12 +1,15 @@
+.container.kt-tabs-component {
+  margin: 0;
+  padding: 0;
+  border-radius: 0px;
```
Comment (ktorii) [inline www/html/css/index2.css]: we don't need this line?

File: www/html/ci/application/controllers/Landing.php (hunk 2 above)
Comment (ktorii) [reply www/html/ci/application/controllers/Landing.php]: This file shouldn't have any changes

**Review by ktorii — CHANGES_REQUESTED** [review]
